static/templates/dashboard/dashboard.html

```diff
@@ -187,4 +187,38 @@
     </div>
 
 </md-data-table>
+</div>
+
+  <md-sidenav md-component-id="right" md-is-locked-open="$mdMedia('min-width: 333px')" class="md-sidenav-right" flex="100">
```

Comment:
put this in it's own div as permanent rather than sidenav as it's supposed to stay there and leave space on the right